docs/user-guide/federation/index.md

```diff
@@ -2,15 +2,68 @@
 title: Federation User Guide
 ---
 
-This guide explains how we can manage multiple Kubernetes clusters using
+This guide explains why and how we can manage multiple Kubernetes clusters using
 federation.
-[Federation proposal](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/proposals/federation.md)
-details the use cases motivating cluster federation.
 
 
 * TOC
 {:toc}
 
+
+## Why federation
+
+Federation makes it easy to manage multiple clusters. It does so by providing 2
+major building blocks:
+
+  * Sync resources across clusters: Federation provides the ability to keep
+    resources in multiple clusters in sync. If you want to ensure that a
+    resource exists in multiple clusters, use federation.
+  * Cross cluster discovery: It provides ability to auto-configure DNS servers
+    and load balancers with backends from all clusters. If you want a global VIP
+    or DNS record to access backends from all clusters, use federation.
+
+Some other use cases that federation enables are:
+
+* High Availability: By spreading load across clusters and auto configuring DNS
+  servers and load balancers, federation minimises the impact of cluster
+  failure.
+* Avoiding provider lock-in: By making it easier to migrate application across
+  clusters, federation prevents cluster provider lock-in.
+
+
+Federation is not helpful unless you have multiple clusters. Some of the reasons
+why you might want multiple clusters are:
+
+* Low latency: Have clusters in many regions so that users are served quickly
+  from a cluster that is close to them.
+* Fault isolation: It might be better to have multiple smaller clusters rather
+  than a single big cluster for fault isolation (for ex: multiple clusters in
+  different availability zones of a cloud provider).
+  [Multi cluster guide](/docs/admin/multi-cluster) has more details on this.
+* Scalability: There are scalability limits to a single kubernetes cluster (this
+  should not be the case for most users).
+* Hybrid cloud: You can have multiple clusters on different cloud providers or
+  on-prem data centers.
+
+
+### Caveats
+
+While there are a lot of attractive use cases for federation, there are also
+some caveats.
+
+* Increased network bandwidth and cost: Federation control plane watches all
+  clusters to ensure that the current state is as expected. This can lead to
+  significant network cost if the clusters are running in different regions on
+  a cloud provider or on different cloud providers.
+* Reduced cross cluster isolation: A bug in federation control plane can impact
```

Comment:
I think that it might be worth expanding upon this a bit further, as it's super-important.  Perhaps mention that the design and implementation in general err on the safety and avoiding multicluster outages.  See for example the recent discussion about defaulting to non-cascading deletion.